PlanningDoc.md

```diff
@@ -0,0 +1,47 @@
+# WRoverPlayground Planning Doc
+
+This is the planning document for the WRoverPlayground exercise repo. This document is a work in progress and anyone can contribute. 
+
+## Topics
+WRoverPlayground needs to encompass the following topics: 
+
+### ROS setup
+TBD
+
+### ROS topics
+Subscriber:
+* Mock navigation data which would be formatting like a lidar, or some coordinate system (would need a lot more math under the hood this way)
+* Robot heading
+* Robot coordinate(?)
+
+Publisher: 
+* Left and right motor powesr
+
+### ROS services
+* Signaling done by mocking status lights
+* Waiting for some amount of time before going to next beacon
+
+### ROS debugging/tooling
+* Using rviz to visualize robot and beacon position
+* Using rqt_plot to monitor topic data
+
+## Steps and instructions
+
+Ideally each step should take one meeting session, so studenets should be able to complete the playground tutorial by the end of the second week. 
+
+### Step 1: Development environment and ROS setup
```

Comment:
Tooling may also move in here so students can use said tools to debug the future code while they're writing it.

Reply:
Not req'd to address.

Reply:
Added in [f16371a](https://github.com/WisconsinRobotics/WRoverPlayground/pull/8/commits/f16371a0f8886efccb3544783daf10e40340e5bf)